server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+require('dotenv').config();
+
+const app = express();
+const config = require('../webpack.config.js');
+
+const compiler = webpack(config);
+
+app.use(webpackDevMiddleware(compiler, {
```

Comment:
Same as above. All webpack dev middleware should only be used in development

Reply:
Going to conditionally use webpack dev middleware depending on environment. Please let me know if that is acceptable.